cmd/juju/commands/ssh_unix_test.go

```diff
@@ -40,8 +46,21 @@ var sshTests = []struct {
 		},
 	},
 	{
-		about: "connect to machine 0 and pass extra arguments",
-		args:  []string{"0", "uname", "-a"},
+		about:      "connect to machine 0 (api v2)",
+		args:       []string{"0"},
+		dialWith:   dialerFuncFor("0.private", "0.public", "0.1.2.3"), // last one set on machine 0 eth0
```

Comment:
It tries to clarify where that "0.1.2.3" came from - it was set by suite.setLinkLayerDevicesAddresses() for suite.m (machine-0). I'll rephrase it to make it clearer.